src/core/core.tooltip.js

```diff
@@ -845,7 +845,7 @@ module.exports = function(Chart) {
 			// Find Active Elements for tooltips
 			if (e.type === 'mouseout') {
 				me._active = [];
-			} else {
+			} else if (e.x > me._chart.chartArea.left) {
```

Comment:
looks good! One last question, should this also check `e.x < me._chart.chartArea.right`? if so, i'd move `me._chart.chartArea` out into a local variable for an improvement to minification

Reply:
Good idea! I added it to the latest PR

Reply:
Right now with the right side of the chart the bug doesn't occur but it could be a good fail safe just in case